### PR TITLE
Add claims model

### PR DESCRIPTION
**Pull request by piafarinella**

### CODE REVIEW

File: app/api_lambda.py [hunk 1]
```diff
@@ -72,3 +76,67 @@ def list(event, context):
         statusCode=result_status,
         body=result_json
     )
+
```
Comment (nicoglp) [inline app/api_lambda.py]: @piafarinella What do you think about create a base class (maybe called `AWSLambda`) where you can assign the target service as an instance property and then use an instance of this class to export the needed functions on this file? 

**Example:**

```
user_lambda = AWSLambda(user_service)
post = user_lambda.post  # Export post lambda function for user_service
```

File: app/config.py [hunk 2]
```diff
@@ -1,10 +1,11 @@
 import os
 
 LOCAL_DYNAMODB_URL = 'http://docker.for.mac.localhost:8000' if 'AWS_SAM_LOCAL' in os.environ else 'http://localhost:8000'
-
+#SQLALCHEMY_DATABASE_URI = 'postgresql+psycopg2://postgres:@172.17.0.2:5432/postgres'
```
Comment (nicoglp) [inline app/config.py]: Maybe you can use 'AWS_SAM_LOCAL' to decide to use Local or Production URL as shown above on `LOCAL_DYNAMODB_URL`

File: template.yaml [hunk 3]
```diff
@@ -56,7 +56,7 @@ Resources:
     Properties:
       Description: 'RESTFull API - GET Method to retrieve list of Users'
       CodeUri: .
-      Handler: app.api_lambda.list
+      Handler: app.api_lambda.list_user
```
Comment (nicoglp) [inline template.yaml]: If you implement the new `AWSLambda` suggested above, you will need to change this config pointing to the new files.

File: app/api_lambda.py (hunk 1 above)
Comment (piafarinella) [reply app/api_lambda.py]: @nicoglp I did this change that you suggested.

**Pull request merged**